### ISSUE TITLE
req: remove app name from toolbar on macOS

### ISSUE THREAD

**Issue by hab0bi** [transmission/transmission#2888]
Can you please delete the app name from the toolbar? It makes the rest of the buttons evanesce:
<img width="627" alt="window" src="https://user-images.githubusercontent.com/82381981/161962717-05875312-924e-4e66-b2be-a78d25a25a97.png">

**Comment by killemov** [comment]
> Can you please delete the app name from the toolbar? It makes the rest of the buttons evanesce: 

... with my small window width. (Right?) IMHO it's the design decision to merge the widow title and the toolbar that is at fault here. The most important button should be at the most left and the least important button at the most right side.

**Comment by kakuhen** [comment]
I haven't inspected the macosx code yet, but something like
```objc
if(@available(macOS 11.0, *)) {
  [mainWindow setStyleMask:NSWindowStyleMaskBorderless];
}
```
in the right spot should solve your problem. We also want to make sure the main window explicitly reports that it can be a key and main window. 

I may try to make a patch for this, but I'm still on 10.15, so I have no reliable way to test the behavior of the new "unified" toolbar present in 11.0 and later versions. Also note that this approach will require Xcode version at least 9, which means building will only work on versions 10.12 and later. Given that [Transmission still builds on version 10.10](https://ports.macports.org/port/transmission/details/), this approach will become a bit more ugly unless we want to drop at least two entire versions of macOS.

Overall, I think the simplest approach to this problem would be to just resize your window. It looks pretty tiny in the picture.

**Comment by GaryElshaw** [comment]
A quick fix. if you open the MainMenu.xib in Xcode, you can uncheck the title.

![mainmenuxib](https://user-images.githubusercontent.com/69029666/173264308-132b70b7-639e-47b1-8dff-3b5349bc8615.png)

**Comment by Coeur** [comment]
Fixed in #3919.
